_posts/2024-08-04-non-academic-problem.md

```diff
@@ -0,0 +1,134 @@
+---
+layout: post
```

Comment:
@leventegego

Add yourself to
https://github.com/versenyprogramozas/vprog-hu/blob/8dbe5ad02d04f949f4fb29918b0283ddfcd5d489/_data/authors.yml#L1-L5
(No need for additional data if you don't want to.)

Then add
```suggestion
layout: post
author: <name>
```